packages/rollup/src/plugins/with-nx/with-nx.ts

```diff
@@ -37,6 +37,7 @@ const image = require('@rollup/plugin-image');
 const json = require('@rollup/plugin-json');
 const copy = require('rollup-plugin-copy');
 const postcss = require('rollup-plugin-postcss');
+const rollupPluginTypescript = require('@rollup/plugin-typescript');
```

Comment:
Minor: Inline this `require` if it's not always used. Maybe it doesn't really matter, but it can add 50ms or so for each inferred project since we always clear require cache when loading configs.